lib/cheetah.rb

```diff
@@ -404,6 +408,17 @@ def run(*args)
 
     private
 
+    def changed_env(options, *args)
```

Comment:
This is a private wrapper for public `run`, invoked recursively from `run`. I don't know exactly why but it feels wrong. 

Also, the specific `run` is being mixed into the generic responsibility of temporarily changing `ENV`.